src/warnings.ml

```diff
@@ -38,6 +38,8 @@ type kind =
   | Predicate_symbol_expected of string
   | Function_symbol_expected of string
   | Pattern_not_exhaustive of string
+  | Patterng_not_exhaustive of string
```

Comment:
Maybe something more self-explanatory like
```suggestion
  | Pattern_guard_not_exhaustive of string
```
would be easier to understand in the code?